docs/docs/components/maskeddatefield.md

```diff
@@ -0,0 +1,156 @@
+---
+title: MaskedDateField
+---
+
+<DocChip chip='shadow' />
+
+<JavadocLink type="foundation" location="com/webforj/component/field/MaskedDateField" top='true'/>
+
+The `MaskedDateField` component is a specialized input field that ensures structured **date entry**. It automatically formats user input according to a specified **date mask**, ensuring consistency across different locales and formats. This is particularly useful for forms requiring standardized date input, such as registrations, scheduling, and booking systems.
+
+## Basics
```

Comment:
That basics section doesn’t seem to add any value. What’s its purpose? What does the user actually learn from it? To me, it just feels like filler.